wbooks/src/app/screens/dashboard/screens/Home/components/SearchBar/index.js

```diff
@@ -0,0 +1,29 @@
+import React from 'react';
+import './styles.css';
+import searchSvg from './assets/search.svg'
+
+class SearchBar extends React.Component {
+
+  handleSubmit = (event) => {
+    event.preventDefault();
+    const value = this.textInput.value;
+    this.props.onSubmit(value);
+  }
+
+  render() {
+    return(
+      <div className="search-bar">
+        <form onSubmit={this.handleSubmit}>
+          <select>
+          </select>
+          <input type="text" placeholder="Buscar..." ref={(textInput) => (this.textInput = textInput)} />
```

Comment:
Move this arrow function to a `instance` method, so we create it once and not on every render.